src/main/java/core/mate/academy/service/RandomService.java

```diff
@@ -0,0 +1,103 @@
+package core.mate.academy.service;
+
+import static core.mate.academy.model.Constants.MAX_BULLDOZER_BLADE_WIDTH;
+import static core.mate.academy.model.Constants.MAX_BULLDOZER_HEIGHT;
+import static core.mate.academy.model.Constants.MAX_BULLDOZER_WEIGHT;
+import static core.mate.academy.model.Constants.MAX_EXCAVATOR_BUCKET_VOLUME;
+import static core.mate.academy.model.Constants.MAX_EXCAVATOR_DIGGING_DEPTH;
+import static core.mate.academy.model.Constants.MAX_TRUCK_CARGO_WEIGHT;
+import static core.mate.academy.model.Constants.MAX_TRUCK_WHEEL_NUM;
+import static core.mate.academy.model.Constants.MIN_BULLDOZER_BLADE_WIDTH;
+import static core.mate.academy.model.Constants.MIN_BULLDOZER_HEIGHT;
+import static core.mate.academy.model.Constants.MIN_BULLDOZER_WEIGHT;
+import static core.mate.academy.model.Constants.MIN_EXCAVATOR_BUCKET_VOLUME;
+import static core.mate.academy.model.Constants.MIN_EXCAVATOR_DIGGING_DEPTH;
+import static core.mate.academy.model.Constants.MIN_TRUCK_CARGO_WEIGHT;
+import static core.mate.academy.model.Constants.MIN_TRUCK_WHEEL_NUM;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Constants;
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RandomService {
+    private final Random random = new Random();
+
+    private int getRandomInt(int from, int to) {
```

Comment:
private methods should follow after public

Reply:
fixed this